### PR TITLE
Adjust titles of Oblique Mercator projection examples

### LINKED ISSUE #2031: More precise names for Oblique Mercator projection examples
GMT seems to support three variants of the Oblique Mercator projection. In the [Projection Table](https://www.pygmt.org/dev/projections/index.html?highlight=Oblique#projection-table) they are distinguished by '1: origin and azim' (`oa|oA` or `Oa|OA` ), '2: two points' (`ob|oB` or `Ob|OB`), and '3: origin and pole' (`oc|oC` or `Oc|OC` ). I think it would be helpful to add these specifications also to the titles of the corresponding examples above. Currently, there are three examples with the same name 'Oblique Mercator'  (see https://www.pygmt.org/dev/projections/index.html?highlight=Oblique#cylindric-projections).

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

Fixes #2031 

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Review by seisman — APPROVED** [review]
Looks good to me. Ping the original author @eemcmullan and reviewers @willschlitzer @michaelgrund in PR https://github.com/GenericMappingTools/pygmt/pull/1251 and see if they have any comments before I push the merge button 24-hours later.

**Review by michaelgrund — APPROVED** [review]
Looks good!

**Pull request merged**
